bindings/python/cconfigspace/__version__.py

```diff
@@ -0,0 +1,3 @@
+VERSION = (0, 0, 0)
+
+__version__ = ".".join(map(str, VERSION))
```

Comment:
`__version__` should be obtained from the new VERSION file.

Reply:
so should I copy the `VERSION` file to `python/cconfigspace/VERSION` when doing `python setup.py install` ?